dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/WebAPI.java

```diff
@@ -168,14 +168,14 @@ public String dateToHTML(Date myDate) {
 	}
 
 	public String dateToHTMLDateTimeRange(Date from, Date to) {
-		return UtilMethods.dateToHTMLDateTimeRange(from, to, GregorianCalendar.getInstance().getTimeZone());
+		return UtilMethods.dateToHTMLDateTimeRange(from, to, APILocator.getCompanyAPI().getDefaultCompany().getTimeZone());
```

Comment:
maybe this code 

<code>
APILocator.getCompanyAPI().getDefaultCompany().getTimeZone()
</code>

should go into a method in a Util class, something like: TimeZoneUtil.getCurrentTimZone()